Index.html

```diff
@@ -0,0 +1,243 @@
+<!doctype html>
+<html lang="en">
+	<head>
+		<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+		<title>bootstrap</title>
+		<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+		<link href="style.css" type="text/css" rel="stylesheet"></link>
+	</head>
+<body>
+	<div class="container">
+		<div class="row bg-white">
+			<div class="col">
+			</div>
+		</div>
+		<div class="row bg-light">
+			<div class="col">
+				<nav class="navbar navbar-expand fixed-top navbar-dark bg-info mx-auto rounded-lg">
+					<a class="navbar-brand" style="font-size:12px;" href="#">
+						<img src="img/logo.png" class="align-bottom rounded-lg" alt="nbaLogo"> Where amazing happens.
+					</a>
+					<ul class="navbar-nav text-center ml-auto mr-1">
+						<li class="nav-item">
+							<a class="nav-link text-white" href="#joinUs">Join Us</a>
+						</li>
+						<li class="nav-item">
+							<a class="nav-link text-white" href="#about">About</a>
+						</li>
+						<li class="nav-item">
+							<a class="nav-link text-white" href="#articles">Articles</a>
+						</li>
+						<li class="nav-item">
+							<a class="nav-link text-white" href="#team">Team</a>
+						</li>
+						<li class="nav-item">
+							<a class="nav-link text-white" href="#feedback">Feedback</a>
+						</li>
+					</ul>
+				</nav>
+			</div >
+		</div>
+		<div class="row border-bottom border-info rounded bg-light" style="min-height: 650px;" id="joinUs">
```

Comment:
general comment to avoid inline styles, in this case it is okay, but in real applications it is usually difficult to support inline styles because of their priority in rendering